docs/about/contributing/CONTRIBUTING.md

```diff
@@ -148,6 +148,32 @@ channel.
 
 - [Frontend styling guide](./frontend.md#styling)
 
+## Pull Requests
+
+We welcome pull requests (PRs) from community members including (but not limited to) open source users, enthusiasts, and enterprise customers.
+
+We will ask that you sign a Contributor License Agreement before we accept any contributions into our repo.
+
+Please keep PRs small and self contained. This allows code reviewers (see below) to focus and fully understand the PR. A good rule of thumb is less than 1000 lines changed. (One exception is a mechanistic refactor, like renaming, that is conceptually trivial but might have a large line count.)
+
+If your intended feature or refactor will be larger than this:
+
+ 1. Open an issue explaining what you intend to build, how it will work, and that you are volunteering to do the development.
```

Comment:
I wonder if we should suggest who to tag on issues like this. Maybe we need a `@maintainers` group or something which review this special class of submission. I wouldn't want these going under the radar.

Reply:
:+1 to a maintainers group or similar.

Reply:
> Maybe "community triage" is the right term for what we want? It comes up very rarely now, so probably throw 1-2 PMs and 1-2 technomancers in a group and call it good?

Works for me 👍 happy to be in that group if needed.